lib/ansible/modules/network/eos/eos_banner.py

```diff
@@ -91,12 +91,13 @@
 from ansible.module_utils.eos import load_config, run_commands
 from ansible.module_utils.eos import eos_argument_spec, check_args
 
+
 def map_obj_to_commands(updates, module):
     commands = list()
     want, have = updates
     state = module.params['state']
 
-    if state == 'absent' and 'text' in have.keys() and have['text']:
+    if state == 'absent' and 'text' in list(have.keys()) and have['text']:
```

Comment:
Looks like list conversion is unnecessary here as the line above is checking if `text` is a key in `have` dictionary. `key in have.keys()` already accomplishes that. Typecasting to list will only increase the complexity. If you disagree please comment.